tools/downloader/tests/representative-models.lst

```diff
@@ -6,3 +6,4 @@ mtcnn-p # Caffe, HTTPS downloads, regex replacement
 octave-densenet-121-0.125 # MXNet, archive unpacking
 resnet-50-pytorch # PyTorch (torchvision)
 single-image-super-resolution-1032 # DLDT
+resnet-50-caffe2 # Caffe2, HTTPS downloads
```

Comment:
In alphabetical order, please. Also, the comments are meant to list the features not covered by any preceding models, so please remove the "HTTPS downloads" bit.